libvast/include/vast/system/actors.hpp

```diff
@@ -419,6 +419,9 @@ using exporter_actor = typed_actor_fwd<
   // Conform to the protocol of the STATUS CLIENT actor.
   ::extend_with<status_client_actor>::unwrap;
 
+using rest_handler_actor = system::typed_actor_fwd<
+  caf::reacts_to<atom::http_request, uint64_t, http_request>>::unwrap;
```

Comment:
I agree it makes sense to create strong typedef's for all the various id's that we're sending as messages, but currently we use uint64/uint32 for everything else as well so that's probably a separate refactoring.